docs/docs/troubleshooting/TROUBLESHOOTING.md

```diff
@@ -262,4 +262,34 @@ This guide contains general tips on how to investigate an application deployment
     kubectl cp atlassian/<pod-name>:/var/atlassian/<application>/logs/<log_files> <local-path>
     ```
 
-   
\ No newline at end of file
+??? tip "How to deal with persistent volume claim destroy failed error?"
+
+    PVC is not able to be destroyed when bounded to pod. Scale down the pod to 0 first before deleting PVC. 
+    
+    `helm upgrade PRODUCT atlassian-data-center/PRODUCT --set replicaCount=0 --reuse-values -n atlassian`
+
+??? tip "How to manually clean up resources when uninstall failed?"
+
+    Sometimes Terraform is unable to destroy resources due to various reasons. This normally happens at EKS level. 
+    One quick solution is to manually delete EKS cluster, and re-run uninstall, so that Terraform will pick up from there. 
+
+    To delete EKS cluster, go to AWS console > EKS service > the cluster you're deploying. 
```

Comment:
```suggestion
    To delete EKS cluster, go to 'AWS console' > 'EKS service' > the cluster you're deploying. 
```

Reply:
I feel AWS console and EKS service is not really a button on the browser, so didn't include quotes.